addons/survey/static/src/question_page/question_page_one2many_field.js

```diff
@@ -48,7 +48,10 @@ class QuestionPageOneToManyField extends X2ManyField {
         const saveRecord = async (record) => {
             await superSaveRecord(record);
             try {
+                const limit = this.list.config.limit;
+                const offset = (Math.ceil(this.list.count / limit) - 1) * limit;
                 await self.props.record.save();
+                await this.list.load({ limit: limit, offset: offset });
```

Comment:
Maybe if we add the method:
```javascript
async loadLastPage(){
  const limit = this.list.config.limit;
  const offset = (Math.ceil(this.list.count / limit) - 1) * limit;
  await this.list.load({ limit: limit, offset: offset });
}
```
Also, we may maybe improve that method to avoid loading the last page if we are already on the last page (I haven't tried). In this case, we could maybe name it ensureLastPageLoaded ?

We can change the method like this:
```javascript
const saveRecord = async (record) => {
    await superSaveRecord(record);
    try {
        await self.props.record.save();
        await this.loadLastPage();
```